app/views/time_based_price_policies/_adjustment_row.html.haml

```diff
@@ -1,22 +1,28 @@
 %td
-  = hidden_field_tag "price_policy_#{price_group.id}[usage_rate]",
-    display_usage_rate(price_group, price_policy),
-    class: "usage_cost"
-  = t("price_policies.adjustment")
-  %br
-  = text_field_tag "price_policy_#{price_group.id}[usage_subsidy]",
-    display_usage_subsidy(price_group, price_policy), size: 8, class: "usage_adjustment"
+  = pp.hidden_field :usage_rate, value: display_usage_rate(price_group, price_policy),
+    class: "js--usageRate"
+  = pp.label :usage_subsidy, t("price_policies.adjustment"), class: "normal-weight"
+  = pp.text_field :usage_subsidy, value: display_usage_subsidy(price_group, price_policy),
+    size: 8,
+    class: "usage_adjustment"
```

Comment:
This class, along with `usage_rate` and `master_usage_cost` are still needed by a different file that I haven't touched which does the per-minute rate calculations.